x/data/server/lookup/table_test.go

```diff
@@ -0,0 +1,256 @@
+package lookup
+
+import (
+	"hash"
+	"hash/fnv"
+	"math"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/store/mem"
+	"github.com/stretchr/testify/require"
+
+	"github.com/tendermint/tendermint/libs/rand"
+)
+
+func TestTable(t *testing.T) {
+	// test default case with good params
+	table, err := NewTable(nil)
+	require.NoError(t, err)
+	testTable(t, table, 5)
```

Comment:
how long it takes to run `1e5` tests? Maybe we can make it smaller as discussed in the PR comments?

Reply:
It's quite quick actually on my machine

Reply:
less than 1sec I believe

Reply:
1 sec for a unit test it's a lot.